### PR TITLE
Update retrieveving-data-and-results.rst to Pt-br

### PR DESCRIPTION
**Pull request by lpj145**

### CODE REVIEW

**Comment by lpj145** [comment]
Why no approve?

**Comment by markstory** [comment]
@lpj145 Sometimes it takes a bit for someone to review and merge changes. We all do this in our evenings and weekends and ask for a bit of patience :smile:

File: pt/orm/retrieving-data-and-resultsets.rst [hunk 1]
```diff
@@ -1,17 +1,1220 @@
...
+
+    // Desabilitando cache explicitamente
+    $article = $articles->get($id, [
+        'cache' => false
+    ]);
+
+
+
+
+``Por padrão o CakePHP possui um sistema interno de cache que viabiliza busca e aumenta a performance - não recomendado desabilatar.``
```
Comment (markstory) [inline pt/orm/retrieving-data-and-resultsets.rst]: Are you sure this whole line should be a code snippet?

Comment (lpj145) [inline pt/orm/retrieving-data-and-resultsets.rst]: yes, in portuguese language, some code explained other than English. Sorry for my patience.

Comment (lpj145) [inline pt/orm/retrieving-data-and-resultsets.rst]: Im translate little by little. CakePHP is wonderful framework, but documentation is no very sure.

Comment (lpj145) [inline pt/orm/retrieving-data-and-resultsets.rst]: please no close this, i used more four hours to this.

Comment (lpj145) [inline pt/orm/retrieving-data-and-resultsets.rst]: please see, which alters the documentation on oficial site CakePHP.

**Comment by markstory** [comment]
Thank you! Hopefully we'll hear from you again soon :smile:

**Comment by lpj145** [comment]
I hope translate more, on little by little i can this.

**Comment by lpj145** [comment]
but, on oficial website nos show documentation, why?

**Comment by markstory** [comment]
@lpj145 Deploys take ~30 minutes.

**Comment by lpj145** [comment]
Im authorize for edit, and merge pull?

**Comment by markstory** [comment]
@lpj145 We give commit access after a number of successful pull requests. Generally all work - even the work done by people with commit access - is done through pull requests anyways.

**Pull request merged**